pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -591,44 +591,44 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 						},
 					},
 					{
-						Record: "num_of_running_virt_controllers",
+						Record: "kubevirt_virt_controllers_up_total",
```

Comment:
We should probably decide if we want plural or singular: `virt_controller` and `virt_api` or `virt_controllers` and `virt_apis`.

Reply:
I'm for the singular

Reply:
singular